doc/nrf/known_issues.rst

```diff
@@ -2119,6 +2124,11 @@ NCSDK-12306: Enabling debug configuration causes usage fault on nRF9160
 Zephyr
 ******
 
+.. rst-class:: v1-9-1 v1-9-0
+
+The time returned by clock_gettime() API is incorrect after one week of uptime
+  The time returned by POSIX clock_gettime() API is incorrect after one week has elapsed since the system booted because of an overflow in the uptime conversion.
```

Comment:
```suggestion
  The time returned by POSIX clock_gettime() API becomes incorrect after one week elapses. This is because the system boots due to an overflow in the uptime conversion.
```

Reply:
Updated as discussed offline.